include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -1174,7 +1147,6 @@ void mergeDistSubproblems(comm::CommunicatorGrid grid,
   sortIndex(i_begin, i_end, pika::make_ready_future(n1), evals, ws.i1, ws.i2);
 
   // --- copy from GPU to CPU if on GPU
-  copy(idx_loc_begin, sz_loc_tiles, evecs, ws_h.evecs);
```

Comment:
just checking with you: is `ws_h.evecs` still needed (and allocated)?

Reply:
Yes, it is needed until distributed permutations are ported to the GPU.